### ISSUE TITLE
It really surprised me a lot

### ISSUE THREAD

**Issue by Breezewrf** [DmitryUlyanov/deep-image-prior#102]
after my "training" on restoration task, I am curious about the loss function -- mse, the parameters including both restoration generated image and ground truth, some minutes after, a function training was finished,
and when I add a watermark different from further one, result as following:

![Screenshot from 2022-09-15 14-27-52](https://user-images.githubusercontent.com/53564343/190330447-dd506200-32ec-48b8-b4ff-c4cdb0457971.png)

last, as far as I am concerned, this algorithm is unrealistic, on the other hand, maybe I have mistaken something because I just met this project yesterday

**Comment by arnavmehta7** [comment]
thats because you need to bitwise_not(mask)
